compiler/rustc_target/src/spec/mod.rs

```diff
@@ -1107,6 +1104,8 @@ impl HasTargetSpec for Target {
     }
 }
 
+type StaticCow<T> = Cow<'static, T>;
```

Comment:
I don't think this adds much. It only saves 2 characters.

Reply:
I asked to add it in https://github.com/rust-lang/rust/pull/95202#discussion_r841230397 because it looks less noisy due to removing punctuation like `'` and `,` even if it's only slightly shorter. (Not a strong opinion.)